### ISSUE TITLE
Regression: Blank line after docstring removed.

### ISSUE THREAD

**Issue by adamtheturtle** [timothycrosley/isort#1275]
Thank you for the software and congratulations on a major release.

--

I have a file like the following:

```python
"""
My docstring
"""

from b import thing
from a import other_thing
```

If I run `isort example.py` I expect this file to change to:

```python
"""
My docstring
"""

from a import other_thing
from b import thing
```

However, it changes to:

```python
"""
My docstring
"""
from a import other_thing
from b import thing
```

with `isort` 5.0.3.

This works as expected with `isort` 4.3.21.

--

Also, the newline is not removed when `isort` does not make sorting changes, so this leaves my codebase inconsistent.

**Comment by adamtheturtle** [comment]
@timothycrosley What a pleasant surprise to see that you have fixed this so quickly 🙏

**Comment by timothycrosley** [comment]
Thank you for reporting!
I'm sorry this issue affected you! A fix has been released with version 5.0.5 of isort.

Thanks!

~Timothy

**Comment by bbugyi200** [comment]
This is still an issue. To replicate:
```bash
isort -a 'from foo import bar' example.py
```
